### PR TITLE
Made user overview admin filterable on user type

### PR DESCRIPTION
**Pull request by Jessseee**
Made user overview admin filterable on user type.

### CODE REVIEW

File: resources/views/users/admin/overview.blade.php [hunk 1]
```diff
@@ -9,19 +9,36 @@
...
+                                <input class="form-check-input" type="radio" name="filter" id="members" value="members" @if($filter == 'members') checked @endif>
+                                <label class="form-check-label" for="members"> Members </label>
+                            </div>
+                            <div class="form-check mr-2">
+                                <input class="form-check-input" type="radio" name="filter" id="pending" value="pending" @if($filter == 'pending') checked @endif>
+                                <label class="form-check-label" for="pending"> Pending </label>
+                            </div>
+                            <div class="form-check">
+                                <input class="form-check-input" type="radio" id="all" name="filter" value="" @if($filter == null) checked @endif>
+                                <label class="form-check-label" for="all"> All </label>
```
Comment (JonaMata) [inline resources/views/users/admin/overview.blade.php:36]: Checkboxes hint to the possibility of multiple filters (for example all members and pending members but not normal users) but this doesn't seem possible with the current implementation.

**Review by JonaMata — CHANGES_REQUESTED** [review]
I would change the filter to either support multiple filters, or to use a dropdown or radio input instead

File: resources/views/users/admin/overview.blade.php (hunk 1 above)
Comment (Jessseee) [reply resources/views/users/admin/overview.blade.php:36]: That is why I used radio buttons

File: app/Http/Controllers/UserAdminController.php [hunk 2]
```diff
@@ -26,20 +26,36 @@ class UserAdminController extends Controller
...
+     * @param Request $request
+     * @return \Illuminate\View\View
      */
     public function index(Request $request)
     {
 
         $search = $request->input('query');
+        $filter = $request->input('filter');
+
+        switch ($filter) {
```
Comment (Jessseee) [inline app/Http/Controllers/UserAdminController.php:38]: @JonaMata Do you have any comments on the queries, I was wondering if perhaps there is an easier or more elegant method?

File: resources/views/users/admin/overview.blade.php (hunk 1 above)
Comment (JonaMata) [reply resources/views/users/admin/overview.blade.php:36]: nvm then

**Review by JonaMata — APPROVED** [review]
LGTM

**Pull request merged**